ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -1,6 +1,8 @@
-<script lang="ts">
+<script lang="typescript">
```

Comment:
Just out of curiosity, what's the difference? So far we've been using `ts` everywhere.
Official docs also mention `ts`: https://svelte.dev/blog/svelte-and-typescript#1_Compiling_TypeScript.

Reply:
Somehow my editor stack doesn't pick it up unless it's the full language name.

Reply:
Whirrd!

I think we should try to keep them consistent, so we should either fix your editor or update the rest of the codebase to use `typescript`.

Reply:
#1032